### ISSUE TITLE
[Visual Requirements- Make Code – Print] button appear as 'hyperlink' in black\white high contrast mode.

### ISSUE THREAD

**Issue by Shalini3457** [microsoft/pxt-microbit#3775]
User Experience:
If button control present on the page appears as hyperlink in Black \white High contrast mode then the end user with low vision who uses this mode will not be able to identify the controls properly and will get disoriented while navigating

Note: User credentials should NOT be included in the bug.

Repro steps:

1. Launch application URL
2. Home page will open.
3. Navigate "New Project" option and select it.
4. Editor screen will open.
5. Navigate to "Settings" option appearing in header section and select it.
6. Navigate to "Print" option and select it.
7. Navigate through all the controls appearing under "Print Code" page
8. Verify the issue.

Actual Result:
button like: 'more setting' appear as 'hyperlink' in black\white high contrast mode.

Expected Result:
button should not appear as hyperlink in black\white high contrast mode. so that low vision user able to see the controls properly and will get disoriented while navigating. 

Note:

This issue does not exists with Firefox and Chrome Browser.

WCAG Reference:
WCAG 4.3.1 - No Disruption of Accessibility Features

Test Environments:
Version: 20H2 (OS Build: 19042.508)
Browser: Edge chromium DEV Build (Version 87.0.654.0)
Screen Reader: Narrator

**Comment by Shalini3457** [comment]
Labels:
#Accessibility; #A11yMAS; #A11ySev2; #HCL; #HCL-MakeCode-New; #WCAG4.1.2; #Win10-Edge(Chromium)

![WCAG4 3 1](https://user-images.githubusercontent.com/72871089/100343417-5725d900-3005-11eb-836b-0bd8de9d4bf7.png)

**Comment by jwunderl** [comment]
This is the browsers print dialog, we do not have control over this.
